test/parallel/test-http2-server-stream-session-destroy.js

```diff
@@ -0,0 +1,48 @@
+// Flags: --expose-http2
+'use strict';
+
+const common = require('../common');
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+const assert = require('assert');
+const h2 = require('http2');
+
+const server = h2.createServer();
+
+server.on(
+  'stream',
+  common.mustCall((stream) => {
+    const errorJSON = {
```

Comment:
Can you name this something else since it's not actually JSON?

Reply:
Done!
Renamed it to errorObj